### PR TITLE
Add octopus release

### PR DESCRIPTION
**Pull request by dsavineau**
Add the 15th ceph release: octopus.

Signed-off-by: Dimitri Savineau <dsavinea@redhat.com>

### CODE REVIEW

**Comment by dsavineau** [comment]
jenkins test pipeline

**Comment by dsavineau** [comment]
jenkins test dev-centos-non_container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-non_container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-non_container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-non_container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-container-update

**Comment by dsavineau** [comment]
jenkins test pipeline

File: tox.ini [hunk 1]
```diff
@@ -299,7 +297,7 @@ commands=
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/{env:PLAYBOOK:site.yml.sample} --extra-vars "\
       delegate_facts_host={env:DELEGATE_FACTS_HOST:True} \
       fetch_directory={env:FETCH_DIRECTORY:{changedir}/fetch} \
-      ceph_stable_release={env:CEPH_STABLE_RELEASE:nautilus} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox.ini]:
```suggestion
      ceph_stable_release={env:CEPH_STABLE_RELEASE:nautilus} \
```

File: tox.ini [hunk 2]
```diff
@@ -253,10 +254,7 @@ setenv=
   dev: UPDATE_CEPH_DOCKER_IMAGE_TAG = latest-master
   dev: UPDATE_CEPH_DEV_BRANCH = master
   dev: UPDATE_CEPH_DEV_SHA1 = latest
-  dev: CEPH_STABLE_RELEASE = nautilus
-
-  switch_to_containers: CEPH_STABLE_RELEASE = nautilus
-  switch_to_containers: CEPH_DOCKER_IMAGE_TAG = latest-nautilus
+  dev: CEPH_STABLE_RELEASE = octopus
```
Comment (guits) [inline tox.ini]:
```suggestion
  dev: CEPH_STABLE_RELEASE = nautilus
```

File: tox.ini [hunk 3]
```diff
@@ -180,7 +181,7 @@ commands=
   ansible-playbook --ssh-extra-args='-F {changedir}/secondary/vagrant_ssh_config' -vv -i {changedir}/secondary/hosts {toxinidir}/{env:PLAYBOOK:site.yml.sample} --extra-vars "\
       ireallymeanit=yes \
       fetch_directory={env:FETCH_DIRECTORY:{changedir}/secondary/fetch} \
-      ceph_stable_release={env:CEPH_STABLE_RELEASE:nautilus} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox.ini]:
```suggestion
```

File: tox.ini [hunk 4]
```diff
@@ -143,7 +144,7 @@ commands=
   ansible-playbook -vv -i {changedir}/hosts-2 --limit osd1 {toxinidir}/add-osd.yml --extra-vars "\
       ireallymeanit=yes \
       fetch_directory={env:FETCH_DIRECTORY:{changedir}/fetch} \
-      ceph_stable_release={env:CEPH_STABLE_RELEASE:nautilus} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox.ini]:
```suggestion
```

File: tox.ini [hunk 5]
```diff
@@ -126,6 +126,7 @@ commands=
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/switch-from-non-containerized-to-containerized-ceph-daemons.yml --extra-vars "\
       ireallymeanit=yes \
       fetch_directory={env:FETCH_DIRECTORY:{changedir}/fetch} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox.ini]:
```suggestion
```

File: tox.ini [hunk 6]
```diff
@@ -90,7 +90,7 @@ commands=
 
   # set up the cluster again
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/{env:PLAYBOOK:site.yml.sample} --extra-vars "\
-      ceph_stable_release={env:CEPH_STABLE_RELEASE:nautilus} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox.ini]:
```suggestion
```

File: tox.ini [hunk 7]
```diff
@@ -79,7 +79,7 @@ commands=
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/{env:PURGE_PLAYBOOK:purge-cluster.yml} --extra-vars "\
       ireallymeanit=yes \
       remove_packages=yes \
-      ceph_stable_release={env:CEPH_STABLE_RELEASE:nautilus} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox.ini]:
```suggestion
```

File: tox.ini [hunk 8]
```diff
@@ -62,7 +62,7 @@ commands=
 
   # set up the cluster again
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/{env:PLAYBOOK:site.yml.sample} --extra-vars "\
-      ceph_stable_release={env:CEPH_STABLE_RELEASE:nautilus} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox.ini]:
```suggestion
```

File: tox.ini [hunk 9]
```diff
@@ -50,7 +50,7 @@ commands=
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/{env:PURGE_PLAYBOOK:purge-cluster.yml} --extra-vars "\
       ireallymeanit=yes \
       remove_packages=yes \
-      ceph_stable_release={env:CEPH_STABLE_RELEASE:nautilus} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox.ini]:
```suggestion
```

File: tox-update.ini [hunk 10]
```diff
@@ -75,13 +75,14 @@ commands=
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/rolling_update.yml --extra-vars "\
       ireallymeanit=yes \
       fetch_directory={env:FETCH_DIRECTORY:{changedir}/fetch} \
+      ceph_stable_release={env:UPDATE_CEPH_STABLE_RELEASE:octopus} \
```
Comment (guits) [inline tox-update.ini]:
```suggestion
```

File: tox.ini [hunk 11]
```diff
@@ -321,7 +319,7 @@ commands=
...
+  all_daemons: ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/{env:PLAYBOOK:site.yml.sample} --extra-vars "delegate_facts_host={env:DELEGATE_FACTS_HOST:True} fetch_directory={env:FETCH_DIRECTORY:{changedir}/fetch} ceph_stable_release={env:CEPH_STABLE_RELEASE:octopus} ceph_docker_registry={env:CEPH_DOCKER_REGISTRY:docker.io} ceph_docker_image={env:CEPH_DOCKER_IMAGE:ceph/daemon} ceph_docker_image_tag={env:CEPH_DOCKER_IMAGE_TAG_BIS:latest-bis-master} ceph_dev_branch={env:CEPH_DEV_BRANCH:master} ceph_dev_sha1={env:CEPH_DEV_SHA1:latest} copy_admin_key={env:COPY_ADMIN_KEY:False}" --extra-vars @ceph-override.json
```
Comment (guits) [inline tox.ini]:
```suggestion
  all_daemons: ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/{env:PLAYBOOK:site.yml.sample} --extra-vars "delegate_facts_host={env:DELEGATE_FACTS_HOST:True} fetch_directory={env:FETCH_DIRECTORY:{changedir}/fetch} ceph_docker_registry={env:CEPH_DOCKER_REGISTRY:docker.io} ceph_docker_image={env:CEPH_DOCKER_IMAGE:ceph/daemon} ceph_docker_image_tag={env:CEPH_DOCKER_IMAGE_TAG_BIS:latest-bis-master} ceph_dev_branch={env:CEPH_DEV_BRANCH:master} ceph_dev_sha1={env:CEPH_DEV_SHA1:latest} copy_admin_key={env:COPY_ADMIN_KEY:False}" --extra-vars @ceph-override.json
```

**Comment by guits** [comment]
> The update scenario now deploy nautilus and update to octopus.

That is incorrect, the update scenario deploys mimic and update to octopus.
Curiously, at the moment nautilus to octopus is failing where mimic to octopus passes. 🙄

**Comment by dsavineau** [comment]
thanks I forgot to update the commit message to reflect that change

**Comment by dsavineau** [comment]
jenkins test dev-centos-non_container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-container-update

File: tox-update.ini (hunk 10 above)
Comment (dsavineau) [reply tox-update.ini]: seems to be mandatory for non container update

**Comment by dsavineau** [comment]
jenkins test dev-centos-non_container-update

**Comment by dsavineau** [comment]
jenkins test dev-centos-container-update

File: tox-update.ini (hunk 10 above)
Comment (guits) [reply tox-update.ini]: not here since we are using `dev_setup.yml`

**Comment by dsavineau** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test pipeline

**Comment by dsavineau** [comment]
jenkins test pipeline

**Pull request merged**